### ISSUE TITLE
Rust "seed" package is held at specific commit revision

### ISSUE THREAD

**Issue by craigmayhew** [craigmayhew/bigprimes.net#46]
This needs updating to use a version tag and not a specific commit hash!

Checking seed v0.4.0 (https://github.com/David-OConnor/seed?rev=c8b3b5d00897e7485e9c0aedbf9eb095d6eb8e96#c8b3b5d0)

**Comment by craigmayhew** [comment]
We can try upgrading once this issue is fixed: https://github.com/David-OConnor/seed/issues/224

**Comment by David-OConnor** [comment]
Were you able to make this work using an init fn like this? 
```rust
fn init(url: Url, orders: &mut impl Orders<Msg>) -> Model {
    let mut model = Model::default();
    update(routes(url).unwrap(), &mut model, orders);
    model
}
```
Stopgap until we make a new release, which has the improved syntax.

**Comment by craigmayhew** [comment]
Implemented in f3941e099bae5272c8822e8842e55b9a73e93b93
